tests/hooks/test_mysql_hook.py

```diff
@@ -236,3 +236,21 @@ def test_bulk_dump(self):
 
     def test_serialize_cell(self):
         self.assertEqual('foo', self.db_hook._serialize_cell('foo', None))
+
+    def test_bulk_load_custom(self):
+        self.db_hook.bulk_load_custom(
+            'table',
```

Comment:
```suggestion
            table='table',
```
What do you think about setting the params explicitly? Makes it more clear I think.